homeassistant/components/tuya/select.py

```diff
@@ -227,9 +227,38 @@
             icon="mdi:layers-outline",
         ),
     ),
+    # Fan
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45vs7vkge
+    "fs": (
+        SelectEntityDescription(
+            key=DPCode.FAN_VERTICAL,
+            name="Vertical Swing Flap Angle",
+            device_class=TuyaDeviceClass.FAN_ANGLE,
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:format-vertical-align-center",
```

Comment:
This entity has a device class and should thus not set an icon.

```suggestion
```

Reply:
Even for custom classes (`TuyaDeviceClass`)? Where can you set the class icon?

Reply:
Oh 🤦 
You're right 👍